src/meta-assistant/generic/message-utils.ts

```diff
@@ -1,6 +1,7 @@
 import { PersonTotalTime } from "src/generated/client/api";
 import { CalculateWorkedTimeAndBillableHoursResponse, DailyCombinedData } from "src/types/meta-assistant";
```

Comment:
```suggestion
import { CalculateWorkedTimeAndBillableHoursResponse, DailyCombinedData } from "src/types/meta-assistant";
``` import type { CalculateWorkedTimeAndBillableHoursResponse, DailyCombinedData } from "src/types/meta-assistant";